### DISCUSSION TITLE
Can't control monitor volume

### DISCUSSION THREAD

**Discussion by ohblue** [MonitorControl/MonitorControl#1703] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

My monitor brand model is: Kuycon G34W
At present, the brightness adjustment works well, but the volume adjustment has no effect.

### Steps to reproduce

1. Plug the speaker’s audio cable into the monitor
2. Select the monitor as the sound output device for Mac

### Expected behavior

I hope the volume can be adjusted normally

### Environment Information (please complete the following information)

```markdown
- macOS version: Sequoia 15.1.1
- Mac model: Mac mini M4 (2024)
- MonitorControl version: v4.2.0 (7048)
- Monitor(s): Kuycon G34W
- Apple Silicon/M1 (yes or no): yes but M4
```

**Comment by Baalakay** [comment]
Same issue on LG monitor. This worked for me. Don't forget to enable DCC settings for sound and restart the app/Mac.
https://github.com/waydabber/BetterDisplay?tab=readme-ov-file#readme

**Comment by RamonBalthazar** [comment]
Same with an LG monitor.

```
- macOS version: Sequoia 15.1.1
- Mac model: MacBook Pro 14-inch 2021
- MonitorControl version: v4.2.0 (7048)
- Apple Silicon/M1 (yes or no): M1 Max
```

**Comment by waydabber** [comment]
Some displays do not support DDC volume adjustments. You might want to try alternate apps like BetterDisplay to see if it works better, but if it's a display firmware issue, it probably won't help.

**Comment by PaviCN** [comment]
Suggestion:
Using full function HDMI / Type-C cable
Reconnected the HDMI cable from the docking station to the monitor's HDMI port
Disabled energy-saving mode in macOS settings
